src/widgets/HospitalizationDeathPlot.tsx

```diff
@@ -153,18 +177,20 @@ export const HospitalizationDeathPlot = ({
   }, [processedData]);
 
   return (
-    <div ref={ref as React.MutableRefObject<HTMLDivElement>} style={{ height: '300px' }}>
-      {width && height && (
-        <GroupedProportionComparisonChart
-          data={processedData}
-          total={total}
-          texts={makeTexts(variantName)[field]}
-          width={width}
-          height={height}
-          extendedMetrics={extendedMetrics}
-          onClickHandler={noopOnClickHandler}
-        />
-      )}
-    </div>
+    <DownloadWrapper name='HospitalizationDeathPlot' rawData={processedData} dataProcessor={dataProcessor}>
```

Comment:
It would be simpler and less abstract if `DownloadWrapper` took `data` (which is directly the output of `dataProcessor(processedData)`), instead of taking `rawData` and `dataProcessor`. You can use `useMemo` to stop `dataProcessor` running on every render.

Reply:
How can we avoid making it update on every data change?

My idea was to make it only run when the person for sure wants to download the data, since it should be quite rare.